src/v0/purchase_orders.ts

```diff
@@ -191,6 +195,21 @@ export class PurchaseOrders extends ThBaseHandler {
     }
   }
 
+  async pdfUri (purchaseOrderId: string): Promise<PurchaseOrdersPdfResponse> {
+    try {
+      const base = this.uriHelper.generateBaseUri(`/${purchaseOrderId}/pdf`)
+      const uri = this.uriHelper.generateUriWithQuery(base)
+
+      const response = await this.http.getClient().get(uri)
+
+      return {
+        data: response.data.results
```

Comment:
Will we actually have multiple files here?
When it's actually single we tend to do this: https://github.com/tillhub/tillhub-sdk-javascript/blob/master/src/v0/purchase_orders.ts#L152